docs/apis-tools/java-client/job-worker.md

```diff
@@ -198,3 +198,40 @@ Since an activated job must now cross two network boundaries - from broker to ga
 We implemented a best-effort mechanism to detect such issues and make the job available again, but this is never guaranteed, as it's not always possible to accurately detect in time that a job did not make it (e.g. network time out, where the job may or may not have made it to the recipient). When this happens, the job will remain in limbo until it times out, at which point it can be pushed out again.
 
 To help ensure accurate detection of client shutdown, make sure to close your job workers gracefully when you're finished with them. This will in turn tell the gateway that the worker is gone, and will help prevent job loss.
+
+## Multi tenancy
+
+You can configure a Job worker pick up jobs belonging to one, or more tenants. When using the builder you can configure
+the tenant(s) it works for. Alternatively, you can configure default tenant(s) on the client. If you configure a
+default, all Job workers you open will work on Jobs for the configured default tenants.
+
+:::note
```

Comment:
:x: I think we should also note that users should ensure that all jobs across all tenants can be processed/handled in the same way. i.e. there's no difference in logic or variables between jobs in different tenants.

Reply:
@koevskinikola This entirely depends on the user, no? If a user has 2 tenants creating jobs with the same job type, then it fully depends on the variables of the different processes whether or not the they are the same for both tenants. It's up to the user to make sure the handler can deal with this.

Not sure what to type here.

Reply:
@remcowesterhoud I guess you have a point. Users always need to ensure that their job workers/handlers can process the jobs with the variables that are provided. This is always true, so it might be redundant to point it out in our docs.